### PR TITLE
New article: relocatable-perl.pod

### PR DESCRIPTION
**Pull request by skaji**
I’m sorry I’m late.
This is an article for #187

### CODE REVIEW

**Review by oalders — APPROVED** [review]
⭐ Looks great!!!

File: 2022/incoming/relocatable-perl.pod [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  use v5.36;
+
+and I'm very excited to use new features.
+
+Unfortunately I don't think too many people use perl v5.36 yet.
+This is because I<system> perls are often out-of-date,
+so people don't have many chances to use the latest perl.
+
+It is true that you, an experienced perl hacker, could build perl v5.36 by yourself.
```
Comment (thibaultduponchelle) [inline 2022/incoming/relocatable-perl.pod:18]:
```suggestion
It is true that you, an experienced Perl hacker, could build perl v5.36 by yourself.
```

File: 2022/incoming/relocatable-perl.pod [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+But people who are not very familiar with perl rarely build perl from source.
```
Comment (thibaultduponchelle) [inline 2022/incoming/relocatable-perl.pod:19]:
```suggestion
But people who are not very familiar with Perl rarely build perl from source.
```

Comment (thibaultduponchelle) [reply 2022/incoming/relocatable-perl.pod:19]: I admit this one sounds strange

**Review by thibaultduponchelle — CHANGES_REQUESTED** [review]

File: 2022/incoming/relocatable-perl.pod [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  Successfully installed 5.36.0.1
+
+=head1 How to use relocatable-perls
+
+You can use relocatable-perls just like your usual perls.
+I have been using relocatable-perls for 8 years and
+don't have any particular problems.
+
+Relocatable-perls come with L<cpanm|https://metacpan.org/pod/App::cpanminus>
+so that you are ready to use CPAN modules!
```
Comment (thibaultduponchelle) [inline 2022/incoming/relocatable-perl.pod:86]: This is great!

**Review by thibaultduponchelle — APPROVED** [review]

**Pull request merged**